pkg/apis/core/v1/helper/helpers.go

```diff
@@ -433,6 +433,26 @@ func GetPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
 	return ""
 }
 
+// GetPersistentVolumeClaimVolumeMode returns VolumeMode. If no VolumeMode was
+// requested, it returns "Filesystem" VolumeMode by default.
+func GetPersistentVolumeClaimVolumeMode(claim *v1.PersistentVolumeClaim) v1.PersistentVolumeMode {
```

Comment:
Now that there is defaulting, these helper methods are not needed and should be removed.

Reply:
They are used in #53385 in the binding logic PR

Reply:
Yes, but now that the fields are defaulted, you no longer need a helper function to default the values.

Reply:
hmmm, they are only defaulted if feature gate is turned on, the binding logic just automatically uses VolumeMode as filesystem

Reply:
ack, removing and will revisit 53385 to make appropriate changes